spec/features/javascript_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'rails_helper'
+
+feature 'JavaScript Tracking', feature: true do
+  before { Raygun.configuration.js_api_key = nil }
+  after { Raygun.configuration.js_api_key = nil }
+
+  it "Includes the Raygun Javascript Middleware" do
+    expect(Rails.application.config.middleware).to include(Raygun::Middleware::JavascriptExceptionTracking)
+  end
+
+  it "No JS snippet is injected" do
+    visit root_path
+
+    expect(page.html).to_not include("<script type=\"text/javascript\">!function(a,b,c,d,e,f,g,h){a.RaygunObject=e,a[e]=a[e]||function(){(a[e].o=a[e].o||[]).push(arguments)},f=b.createElement(c),g=b.getElementsByTagName(c)[0],f.async=1,f.src=d,g.parentNode.insertBefore(f,g),h=a.onerror,a.onerror=function(b,c,d,f,g){h&&h(b,c,d,f,g),g||(g=new Error(b)),a[e].q=a[e].q||[],a[e].q.push({e:g})}}(window,document,\"script\",\"//cdn.raygun.io/raygun4js/raygun.min.js\",\"rg4js\");</script></head>")
```

Comment:
Probably better to assert something simpler here, maybe just checking for the presence of `raygun.min.js`?